OpenTelemetry.sln

```diff
@@ -141,7 +135,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Instrumentati
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestApp.AspNetCore.2.1", "test\TestApp.AspNetCore.2.1\TestApp.AspNetCore.2.1.csproj", "{7CDA3F85-EA6F-45C4-B432-2559303355D4}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenTelemetry.Exporter.ZPages.Tests", "test\OpenTelemetry.Exporter.ZPages.Tests\OpenTelemetry.Exporter.ZPages.Tests.csproj", "{98F9556B-116F-49B5-9211-BB1D418446FF}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Exporter.ZPages.Tests", "test\OpenTelemetry.Exporter.ZPages.Tests\OpenTelemetry.Exporter.ZPages.Tests.csproj", "{98F9556B-116F-49B5-9211-BB1D418446FF}"
```

Comment:
Curious - is this `.sln` file useful for the project at all or not? I'm guessing only VS IDE users would use it (and they can use the proj file without sln, too)?

Reply:
I ask this question since diff the project GUIDs each time we change the structure doesn't sound like something worthy?

Reply:
VS IDE need sln files. Don't think it understand the .proj file.

Reply:
I see. Do we need to maintain a VS IDE file at all?

Reply:
For example, for me, if I try to work using VS Code, i can't even compile. it shows more than 2k errors...

Reply:
That seems to be a bigger issue (that we're forcing community folks to use VS IDE). Not a blocker for this PR though.
BTW, the CI pipeline is not relying on VS IDE to build 😄

Reply:
I still use VS 😭

Reply:
i will try to find out why does that issues happen.

Reply:
@reyang , just pushed one change to show less errors in vscode problems tab.